src/OrchardCore/OrchardCore.Contents.Core/SiteServiceExtensions.cs

```diff
@@ -0,0 +1,17 @@
+using System.Threading.Tasks;
+using OrchardCore.ContentManagement;
+using OrchardCore.Entities;
+
+namespace OrchardCore.Settings;
+
+public static class SiteServiceExtensions
+{
+    /// <summary>
+    /// Gets an instance of the specified settings if it exists.
+    /// </summary>
+    /// <param name="siteService">The site service.</param>
+    /// <param name="name">The entry name.</param>
+    /// <returns>An <see cref="ContentItem" /> instance that matches the given name, if one exists.</returns>
+    public static async Task<ContentItem> GetCustomSettingsAsync(this ISiteService siteService, string name)
```

Comment:
Line 52 in `SeoMetaSettingsHandler` can be changed to this.

Reply:
done

Reply:
@Piedone actually I undid this change. Down on line 111 we use the SiteSettings to get the SiteName so it's better to just use .As<> and we don't need to add dependency on Contents.Core.